src/components/PostForm.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <form action="">
```

Comment:
do we need form?

Reply:
the code will work without a form, but I know that if we have an input, we must write it in the form or it's not so important ?

Reply:
it's not important. Form is for having multiple inputs in one place and have control of them by submit button. But you don't have something like that

Reply:
okay thank i will remove it